charts/stable/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -29,14 +27,22 @@ spec:
         - "$(DB_PASSWORD)"
         {{- if eq .Values.schemaLoading.database "cassandra" }}
         - "--cassandra"
+        - "-h"
+        - "{{ .Values.schemaLoading.contactPoints }}"
         - "-n"
         - "NetworkTopologyStrategy"
         - "-R"
         - "{{ .Values.schemaLoading.cassandraReplicationFactor }}"
         {{- else if eq .Values.schemaLoading.database "cosmos" }}
         - "--cosmos"
+        - "-h"
+        - "{{ .Values.schemaLoading.contactPoints }}"
         - "-r"
         - "{{ .Values.schemaLoading.cosmosBaseResourceUnit }}"
+        {{- else if eq .Values.schemaLoading.database "dynamo"}}
+        - "--dynamo"
+        - "--region"
+        - "{{ .Values.schemaLoading.contactPoints }}"
```

Comment:
For not, it is fine, but I feel we should unify the schema tool's parameter for `contractPoints`.